core/.js/src/main/scala/io/udash/core/Defaults.scala

```diff
@@ -1,9 +1,17 @@
 package io.udash.core
 
-/** Creates view with [[io.udash.core.EmptyPresenter]]. Useful for static views. */
-class StaticViewFactory[S <: State](viewCreator: () => View) extends ViewFactory[S] {
+/**
+ * Creates view with [[io.udash.core.EmptyPresenter]]. Used for static views.
+ *
+ * By default, instances of this class are compared by class name to prevent rerendering of static views.
+ * This behaviour can be opted out of by overriding equals/hashCode.
+ **/
+abstract class StaticViewFactory[S <: State](viewCreator: () => View) extends ViewFactory[S] {
   override def create(): (View, EmptyPresenter.type) =
     (viewCreator(), EmptyPresenter)
+
+  override def equals(other: Any): Boolean = getClass.equals(other.getClass)
```

Comment:
That's a rather unlikely case, but `other.getClass` may throw a NullPointerException here.